sys/linux/seccomp_386.const

```diff
@@ -1,4 +1,6 @@
 # AUTOGENERATED FILE
+# O_CLOEXEC is not set
```

Comment:
Looks like seccomp.txt is missing the header file that defines O_CLOEXEC.

Reply:
Thanks for the catch! Surprised that it can still work though.